sources/packages/backend/apps/api/src/route-controllers/program-year/_tests_/e2e/program-year.students.controller.getProgramYears.e2e-spec.ts

```diff
@@ -0,0 +1,157 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import { ProgramYear } from "@sims/sims-db";
+import * as request from "supertest";
+import { DataSource } from "typeorm";
+
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getStudentToken,
+  FakeStudentUsersTypes,
+} from "../../../../testHelpers";
+
+describe("ProgramYearStudentsController(e2e)-getProgramYears", () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    appDataSource = dataSource;
+    app = nestApplication;
+  });
+
+  it("Should get the list of the available program years.", async () => {
+    // Arrange
+    const endpoint = "/students/program-year/options-list";
+    const studentToken = await getStudentToken(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(studentToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .then((response) => {
+        const programYearList = response.body;
+        expect(programYearList).toHaveLength(3);
+        expect(programYearList).toEqual(
+          expect.arrayContaining([
+            expect.objectContaining({
+              id: 1,
+              description:
+                "(2021-2022) - Study starting between August 01, 2021 and July 31, 2022",
+            }),
+            expect.objectContaining({
+              id: 2,
+              description:
+                "(2022-2023) - Study starting between August 01, 2022 and July 31, 2023",
+            }),
+            expect.objectContaining({
+              id: 3,
+              description:
+                "(2023-2024) - Study starting between August 01, 2023 and July 31, 2024",
+            }),
+          ]),
+        );
+      });
+  });
+
+  it("Should not include inactive program years in the list.", async () => {
+    // Arrange
+    const programYearRepo = appDataSource.getRepository(ProgramYear);
+    const programYear = await programYearRepo.findOneBy({ id: 2 });
```

Comment:
Changing the current program years causes side effects in other tests.
This would be the same case of https://github.com/bcgov/SIMS/pull/1803/files#r1137781213